### ISSUE TITLE
Pulumi checksum files are different in github and pulumi.com

### ISSUE THREAD

**Issue by amkartashov** [pulumi/pulumi#9416]
### What happened?

Checksum file https://get.pulumi.com/releases/sdk/pulumi-3.29.1-checksums.txt is wrong and does not match https://github.com/pulumi/pulumi/releases/download/v3.29.1/pulumi-3.29.1-checksums.txt

### Steps to reproduce

```
$ wget https://get.pulumi.com/releases/sdk/pulumi-3.29.1-checksums.txt
$ wget https://github.com/pulumi/pulumi/releases/download/v3.29.1/pulumi-3.29.1-checksums.txt

$ diff -u pulumi-3.29.1-checksums.txt*
--- pulumi-3.29.1-checksums.txt 2022-04-14 11:17:36.000000000 +0400
+++ pulumi-3.29.1-checksums.txt.1       2022-04-14 06:37:03.000000000 +0400
@@ -1,5 +1,5 @@
 3687eb599131610b31c8993e3493401fa54f1e8577f208b04ba88048404b1f52  pulumi-v3.29.1-windows-x64.zip
-54920bbba46613aef7b61769f477a3b8c9a667929001743ae827b29d959f1d32  pulumi-v3.29.1-linux-x64.tar.gz
-59be21c9083d40102867736ad860b9897f180413e21b0919b171e5329594940e  pulumi-v3.29.1-darwin-arm64.tar.gz
-ae29823fa5016005b0229545e3b619470a12f207d96af796ed50df10e5b4a15f  pulumi-v3.29.1-linux-arm64.tar.gz
-af180d963fcc68c521cc6df47a62f50599a990c8426a924d1d8f5a00ed710c4d  pulumi-v3.29.1-darwin-x64.tar.gz
+5cab590e59cfd01505e903be3ead8df24b78eefccce0c2a68e37771926c959da  pulumi-v3.29.1-linux-arm64.tar.gz
+5dd8e5f19173fd284f94ad3a12c3dd417cc57f7fac39d6be82947832b13e5330  pulumi-v3.29.1-darwin-x64.tar.gz
+5f360484020169eea3e14bdb22be7acb29829c9082913b0c302e70ffff7aa0de  pulumi-v3.29.1-linux-x64.tar.gz
+633be57052b196c2cca67d1911c22413354d3b1c3608dbbf97489d5859808059  pulumi-v3.29.1-darwin-arm64.tar.gz
```

### Expected Behavior

should be the same

### Actual Behavior

different

### Additional context

similar issue for prev version https://github.com/pulumi/pulumi/issues/9063

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by mikhailshilkov** [comment]
@cnunciato What have we done to fix #9063? Do we know the reason?

**Comment by justinvp** [comment]
I'm going to close this stale issue. I've verified the checksums match for the latest release.

```
$ wget https://get.pulumi.com/releases/sdk/pulumi-3.130.0-checksums.txt
$ wget https://github.com/pulumi/pulumi/releases/download/v3.130.0/pulumi-3.130.0-checksums.txt
$ diff -u pulumi-3.130.0-checksums.txt*
```
